src/Admin/Input/Input.php

```diff
@@ -142,6 +158,34 @@ public function set_value( $value ): InputInterface {
 		return $this;
 	}
 
+	/**
+	 * @param bool $value
+	 *
+	 * @return InputInterface
+	 */
+	public function set_readonly( $value = false ) {
+		$this->is_readonly = $value;
+
+		return $this;
+	}
+	/**
+	 * @param bool $value
+	 *
+	 * @return InputInterface
+	 */
+	public function set_disabled( $value = false ) {
+		$this->is_disabled = $value;
+
+		return $this;
+	}
```

Comment:
WDYT of adding the types in the arguments?

Reply:
Another concern I have is that `disabled` is the right name to use for a hidden field. As in HTML disabled inputs are still readable. So I would suggest `is_hidden` / `set_hidden` but not a blocker tho,